full-stack-tests/native-app/src/backend/backend.ts

```diff
@@ -52,6 +86,16 @@ async function init() {
   iModelHost.concurrentQuery.concurrent = 2;
   iModelHost.concurrentQuery.pollInterval = 5;
   iModelHost.cacheDir = path.join(__dirname, "out");
-  await ElectronHost.startup({ electronHost: { rpcInterfaces }, iModelHost });
+
+  await ElectronHost.startup({
+    electronHost: {
+      ipcHandlers: [TestIpcHandler],
+      authConfig: {
+        clientId: "testapp", redirectUri: "", scope: "",
+      },
+    },
+    iModelHost,
+  });
 }
+
 module.exports = init();
```

Comment:
@wgoehrig This seems unnecessary/obscure to me. Isn't there a better way to get an init function called?